lexicon.txt

```diff
@@ -51,12 +51,13 @@ nb
 nextkeyvaluepair
 noninfringement
 nul
```

Comment:
Can we delete this misspelled word? It looks like its not existent in the library.

Reply:
NUL is the ASCII name for '\0'.  It is in the comment for skipEscape().